wagtailmenus/templatetags/menu_tags.py

```diff
@@ -608,8 +645,17 @@ def prime_menu_items(
             menuitem.link_page = page
 
         # Both `Page` and `MenuItem` objects have a `relative_url` method
-        # we can use to calculate a value for the `href` attribute.
-        setattr(item, 'href', item.relative_url(current_site))
+        # and a `get_full_url` method (added in Wagtail 1.11) or `full_url`
```

Comment:
Talking about `Page` and `MenuItem` objects in the same comment here is a little confusing now, as the wagtail version stuff applies to one and not the other. I think we should probably remove this all together, and replace it with two shorter comments.

And one above what is currently at line `652`:

> Pages only have `get_full_url` from Wagtail 1.11 onwards

And one above what is currently at line `655`:

> Fallback for Wagtail versions prior to 1.11